acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -43,6 +43,29 @@ def expect_file_on_host_to_contain(host, file, expected, seconds=30)
   end
 end
 
+# Show the logs of the broker and agents - useful to see why a test failed
+def show_pcp_logs
+  puts "---- Broker log -----"
```

Comment:
Ideally, the beaker logger instance should always be used instead of puts. I think in practice, in our current CI, it makes no difference. PCP-365 mentions doing a sweep through the acceptance to remove all puts and print statements (i.e. I'm not sure there's value in changing this immediately; we can do the more general tidy-up later)